.kokoro/datacatalog/cloud-client/datacatalog.cfg

```diff
@@ -12,14 +12,20 @@ env_vars: {
     value: "github/nodejs-docs-samples/.kokoro/build.sh"
 }
 
-# Specify the project that contains a valid dataset.
+# Specify the project that datacatalog will run on.
 env_vars: {
 	key: "GCLOUD_PROJECT"
```

Comment:
`GCLOUD_PROJECT` isn't a whitelisted environment variable. Please change to `PROJECT`.

Reply:
done, I also added the common.cfg using one as example, please verify if it's ok.